htdocs/core/lib/ajax.lib.php

```diff
@@ -357,12 +357,14 @@ function ajax_dialog($title,$message,$w=350,$h=150)
  * @param  	int		$minLengthToAutocomplete	Minimum length of input string to start autocomplete
  * @param	int		$forcefocus					Force focus on field
  * @param	string	$widthTypeOfAutocomplete	'resolve' or 'off'
+ * @param	int		$htmlid						Html id to use instead of htmlname
  * @return	string								Return html string to convert a select field into a combo, or '' if feature has been disabled for some reason.
  */
-function ajax_combobox($htmlname, $events=array(), $minLengthToAutocomplete=0, $forcefocus=0, $widthTypeOfAutocomplete='resolve')
+function ajax_combobox($htmlname, $events=array(), $minLengthToAutocomplete=0, $forcefocus=0, $widthTypeOfAutocomplete='resolve',$htmlid='')
```

Comment:
Why adding a new param here. If it is set, value is used instead of htmlname. Why don't you provide value of htmlid directly into param htmlname ? See command of param, it can be a class or an html id.